lib/Fmt.mli

```diff
@@ -124,22 +124,17 @@ val break_unless_newline : int -> int -> t
 
 (** Conditional on breaking of enclosing box ----------------------------*)
 
+type behavior = Fit | Break
+
 val fits_breaks :
-     ?force_fit_if:bool
-  -> ?force_break_if:bool
-  -> ?hint:int * int
-  -> ?level:int
-  -> string
-  -> string
-  -> t
+  ?force:behavior -> ?hint:int * int -> ?level:int -> string -> string -> t
```

Comment:
Just to check: is this common enough that `?force` is not passed? in other word, does it make sense to keep the parameter optional?

Reply:
Yes most of the times we don't pass this parameter, I will try to reduce the usage of this parameter in the future whenever possible